### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by vorot93**
* Simplified lead section layout, removed old logo.
* Fixed project name and links.
* Removed link to currently absent security policy.
* Moved Discord badge into the lead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
-![Open Ethereum](docs/logo-parity-ethereum.svg)
+# OpenEthereum
 
-<h2 align="center">The Fastest and most Advanced Ethereum Client.</h2>
+The Fastest and most Advanced Ethereum Client.
```
Comment (dvdplm) [inline README.md]: How about "A fast and feature rich multi-network Ethereum Client"?

File: README.md [hunk 2]
```diff
@@ -35,19 +42,19 @@
 
 ## 2. Technical Overview <a id="chapter-002"></a>
 
-Open Ethereum's goal is to be the fastest, lightest, and most secure Ethereum client. We are developing Open Ethereum using the sophisticated and cutting-edge **Rust programming language**. Open Ethereum is licensed under the GPLv3 and can be used for all your Ethereum needs.
+OpenEthereum's goal is to be the fastest, lightest, and most secure Ethereum client. We are developing OpenEthereum using the sophisticated and cutting-edge **Rust programming language**. OpenEthereum is licensed under the GPLv3 and can be used for all your Ethereum needs.
```
Comment (dvdplm) [inline README.md]:
```suggestion
OpenEthereum's goal is to be the fastest, lightest, and most secure Ethereum client. We are developing OpenEthereum using the **Rust programming language**. OpenEthereum is licensed under the GPLv3 and can be used for all your Ethereum needs.
```

File: README.md [hunk 3]
```diff
@@ -35,19 +42,19 @@
...
+If you run into problems while using OpenEthereum, check out the [old wiki for documentation](https://wiki.parity.io/), feel free to [file an issue in this repository](https://github.com/openethereum/openethereum/issues/new), or hop on our [Discord](https://discord.io/openethereum) chat room to ask a question. We are glad to help!
 
-You can download Open Ethereum's latest release at [the releases page](https://github.com/OpenEthereum/open-ethereum/releases) or follow the instructions below to build from source. Please, mind the [CHANGELOG.md](CHANGELOG.md) for a list of all changes between different versions.
+You can download OpenEthereum's latest release at [the releases page](https://github.com/openethereum/openethereum/releases) or follow the instructions below to build from source. Please, mind the [CHANGELOG.md](CHANGELOG.md) for a list of all changes between different versions.
```
Comment (dvdplm) [inline README.md]:
```suggestion
You can download OpenEthereum's latest release at [the releases page](https://github.com/openethereum/openethereum/releases) or follow the instructions below to build from source. Read the [CHANGELOG.md](CHANGELOG.md) for a list of all changes between different versions.
```

File: README.md [hunk 4]
```diff
@@ -109,26 +116,26 @@ This always compiles the latest nightly builds. If you want to build stable, do
...
 #### Using `systemd` service file
 
-To start Open Ethereum as a regular user using `systemd` init:
+To start OpenEthereum as a regular user using `systemd` init:
 
 1. Copy `./scripts/parity.service` to your
 `systemd` user directory (usually `~/.config/systemd/user`).
 2. Copy release to bin folder, write `sudo install ./target/release/parity /usr/bin/parity`
-3. To configure Open Ethereum, write a `/etc/parity/config.toml` config file, see [our old wiki](https://paritytech.github.io/wiki/Configuring-Parity) for details.
+3. To configure OpenEthereum, write a `/etc/parity/config.toml` config file, see [our old wiki](https://paritytech.github.io/wiki/Configuring-Parity) for details.
```
Comment (dvdplm) [inline README.md]:
```suggestion
3. To configure OpenEthereum, write a `/etc/openethereum/config.toml` config file, see [our old wiki](https://paritytech.github.io/wiki/Configuring-Parity) for details.
```

I don't think that path is special but I'm not 100% sure!

File: README.md [hunk 5]
```diff
@@ -290,9 +297,9 @@ Caching, Importing Blocks, and Block Information
 
 </p></details>
 
-### Contributing to documentation for Open Ethereum packages
+### Contributing to documentation for OpenEthereum packages
```
Comment (dvdplm) [inline README.md]: I think this whole paragraph can be deleted. What's the point in saying "document rust code using the rust documentation best practices"?

File: README.md [hunk 6]
```diff
@@ -318,31 +325,24 @@ Example (generic documentation comment):
...
+In addition to the OpenEthereum client, there are additional tools in this repository available:
 
-- [evmbin](./evmbin) - Open Ethereum EVM Implementation.
-- [ethstore](./accounts/ethstore) - Open Ethereum Key Management.
-- [ethkey](./accounts/ethkey) - Open Ethereum Keys Generator.
+- [evmbin](./evmbin) - OpenEthereum EVM Implementation.
+- [ethstore](./accounts/ethstore) - OpenEthereum Key Management.
+- [ethkey](./accounts/ethkey) - OpenEthereum Keys Generator.
 
 The following tool is available in a separate repository:
```
Comment (dvdplm) [inline README.md]:
```suggestion
The following tools are available in a separate repository:
```

File: README.md (hunk 4 above)
Comment (niklasad1) [reply README.md]: Yes, it is not used unless `--config` is used

Comment (vorot93) [reply README.md]: If it's not used, then we should remove this point?

Comment (niklasad1) [reply README.md]: I was wrong, it is used if the `BASE_PATH` is `/etc/openethereum` and can be overwritten by `--config ..../some/some.toml` but I think the following with be sufficent:

> To configure OpenEthereum, see [our old wiki](https://paritytech.github.io/wiki/Configuring-Parity) for details.

**Pull request merged**